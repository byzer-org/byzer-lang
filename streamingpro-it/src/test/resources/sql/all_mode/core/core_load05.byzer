--%comparator=tech.mlsql.it.IgnoreResultComparator

load excel.`/excel/test-excel.xlsx` where header="true" and dataAddress="'Sheet2'!A20:B21" and addColorColumns="true" as excel_demo;

!assert excel_demo ''':id == 1''' "Failed to load excel!";
!assert excel_demo ''':title == 10''' "Failed to load excel!";
!assert excel_demo ''':id_color == "FFC6EFCE"''' "Failed to load excel!";
!assert excel_demo ''':title_color == "FFFFC7CE"''' "Failed to load excel!";