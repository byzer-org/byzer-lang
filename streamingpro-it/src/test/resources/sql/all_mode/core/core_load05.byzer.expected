id,title,name,remark,id_color,title_color,name_color,remark_color
1,10,张三,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
2,9,李四,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
3,8,王二麻子,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
4,7,张三,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
5,6,李四,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
6,5,王二麻子,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
7,4,张三,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
8,3,李四,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
9,2,王二麻子,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
10,1,张三,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
11,0,李四,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
12,-1,王二麻子,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
13,-2,张三,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2
14,-3,李四,asd,FFC6EFCE,FFFFC7CE,FFFFEB9C,FFF2F2F2